keyboards/philoandgunge/nyaaaah54_rp2040/README.md

```diff
@@ -0,0 +1,25 @@
+# nyaaaah54_rp2040
+
+![nyaaaah54_rp2040](https://i.imgur.com/nyaaaah54_rp2040.jpg)
+
+A trashy, handwired hotswap-ish monoblock ergo. Made from salvaged & upcycled parts and materials from the all-knowing trash heap in the back alleyway.
```

Comment:
> handwired

Project **must** be relocated be within the `keyboards/handwired` directory.

Reply:
Link doesn't work either